pkg/northbound/diags/diags.proto

```diff
@@ -123,6 +103,10 @@ message ListDeviceChangeRequest {
     // subscribe indicates whether to subscribe to events (e.g. ADD, UPDATE, and REMOVE) that occur
     // after all devices have been streamed to the client
     bool subscribe = 1;
+
+    // option to specify a specific device change - if blank or '*' then select all
+    // Can support `*` (match many chars) or '?' (match one char) as wildcard
+    string changeid = 2;
```

Comment:
FYI you can use gogo to generate a Go struct field that's properly cased:
```go
string change_id = 2 [(gogoproto.customname) = "ChangeID"];
```
and also properly typed
```go
string change_id = 2 [(gogoproto.customname) = "ChangeID", casttype = "github.com/onosproject/onos-config/pkg/types/change/network.ID"];
```

Reply:
Thanks  - will do that